pylint/checkers/refactoring/refactoring_checker.py

```diff
@@ -464,9 +464,9 @@ class RefactoringChecker(checkers.BaseTokenChecker):
             "The literal is faster as it avoids an additional function call.",
         ),
         "R1735": (
-            "Consider using {} instead of dict()",
+            "Consider using '%s' instead.",
```

Comment:
Yet another change of mind for clarity, let's hear what other maintainer think before applying :) 

```suggestion
            "Consider using '%s' instead of a call to 'dict'.",
```